test/helpers/utils.js

```diff
@@ -40,18 +40,27 @@ exports.checkOutput = function (f, argv, cb) {
 
     f()
   } else {
-    result = f()
+    try {
+      result = f()
+    } finally {
+      reset()
```

Comment:
If `f()` doesn't throw, `reset()` is called twice. But I guess twice is better than never. ;)